packages/client/hmi-client/src/services/event.ts

```diff
@@ -21,7 +22,7 @@ async function get(
 		const response = await API.get(`/events?${toQueryString(params)}`);
 		return response?.data ?? [];
 	} catch (e) {
-		console.log('Error fetching events: ', e);
+		logger.error(`Error fetching event: ${e}`);
```

Comment:
Is there a significance in using ` here as a quotation mark?  I see a mix of that and ' in this PR, and in some cases like this one, turning one into the other.

Reply:
This is just es7 string literals?

Reply:
It's because we need them for [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)